snowpack/src/sources/local.ts

```diff
@@ -430,7 +434,7 @@ export default {
           });
           logger.debug(`${lineBullet} ${packageFormatted} (ssr) DONE`);
         }
-        const dependencyFileLoc = path.join(installDest, newImportMap.imports[spec]);
+        const dependencyFileLoc = path.join(installDest, installResult.importMap.imports[spec]);
```

Comment:
This wasn’t a necessary change, but I wanted to prevent `newImportMap` from shadowing the same variable name in the parent scope, also defined on L344. Could be an annoying mistake down the line.

Reply:
yea this is the one thing I always miss in eslint, I wish there was a tsconfig option to warn on variable shadowing

Reply:
We can set up ESLint! 😏 We could just add one-rule-at-a-time, starting w/ this.

Reply:
https://www.youtube.com/watch?v=F_XaIuw6K6Q